R/metadata_tables.R

```diff
@@ -152,7 +152,7 @@ get_examples <- function(ds, columns) {
       \(x) unique(ds$postgres_export[, x, drop = F]) %>%
         head(3) %>%
         dplyr::pull(x) %>%
-        paste(sep = " ", collapse = "; ")
```

Comment:
That is the point us using paste0 with a space separator here? When trying it on iris I get the following:

```r
> iris %>% head(3) %>% dplyr::pull(Species) %>% paste(sep=" ", collapse=",")
[1] "setosa,setosa,setosa"
> iris %>% head(3) %>% dplyr::pull(Species) %>% paste0(sep=" ", collapse=",")
[1] "setosa ,setosa ,setosa "
```
I don't think we want to add spaces _before_ the value (or even at all), do we?

Reply:
Good catch, thank you. Fixed it.